sydent/sms/openmarket.py

```diff
@@ -67,10 +67,7 @@ async def sendTextSMS(
         Sends a text message with the given body to the given MSISDN.
 
         :param body: The message to send.
-        :type body: str
```

Comment:
I think the type hint here might actually be wrong. It looks like the only caller to this does use a string as `body`.

mypy will complain about `body` being redefined, you'll need to update the variable name below to something new, most likely.